client/components/domains/register-domain-step/index.jsx

```diff
@@ -78,7 +77,7 @@ const domains = wpcom.domains();
 const SUGGESTION_QUANTITY = 10;
 const INITIAL_SUGGESTION_QUANTITY = 2;
 
-let searchVendor = 'domainsbot';
+let searchVendor = 'group_1';
```

Comment:
These are very confusing. Maybe we should create names with meaning, and still not explicitly say what. :)